tests/test_background.py

```diff
@@ -602,8 +602,8 @@ def testBadRows(self):
             self.assertEqual(
                 np.mean(bkgdImage[0:100, 0:100].array), initialValue)
             if debugMode:
-                afwDisplay.Display(frame=frame).mtv(bkgdImage, title=self._testMethodName + " bkgdImage: " +
-                                                    interpStyle.__str__())
+                afwDisplay.Display(frame=frame).mtv(bkgdImage, title=self._testMethodName + " bkgdImage: "
+                                                    + interpStyle.__str__())
```

Comment:
consider an f-string here to allow the both of the `+` to go away.

Reply:
Note that an f-string does make the line very long (in fact I have to move title=f"..." to the next line). But it squeaks by.

Reply:
You can split the f-string over two lines like any other explicit quoted string.  You don't need the `+` to do it though.